Mathlib/Analysis/SpecialFunctions/PolarCoord.lean

```diff
@@ -185,4 +200,11 @@ protected theorem integral_comp_polarCoord_symm {E : Type*} [NormedAddCommGroup
     measurableEquivRealProd.symm.measurableEmbedding, ← integral_comp_polarCoord_symm]
   rfl
 
+protected theorem lintegral_comp_polarCoord_symm (f : ℂ → ℝ≥0∞) :
+    (∫⁻ p in polarCoord.target, ENNReal.ofReal p.1 • f (Complex.polarCoord.symm p)) =
+      ∫⁻ p, f p := by
+  rw [← (volume_preserving_equiv_real_prod.symm).lintegral_comp_emb
+    measurableEquivRealProd.symm.measurableEmbedding, ← lintegral_comp_polarCoord_symm]
+  rfl
```

Comment:
Why do you need the rfl? Missing lemma?

Reply:
Indeed. I added the missing lemma. Thanks!

Still, not sure about the name though so any suggestion for a better name is welcome.